extensions/amp-form/amp-form.md

```diff
@@ -23,7 +23,12 @@ limitations under the License.
   </tr>
   <tr>
     <td width="40%"><strong>Availability</strong></td>
-    <td>Stable<br>(<a href="#custom-validations">Custom Validation still experimental - See below</a>)</td>
+    <td>Stable<br>Experimental Features:
```

Comment:
Can you add the link to https://www.ampproject.org/docs/reference/experimental for the word "Experimental".